### PR TITLE
starting with adding cities and teams to db and endpoints

### PR DESCRIPTION
**Pull request by rwwarren**
added stuff for cities and teams connection

### CODE REVIEW

File: src/main/resources/db/migration/V2__Create_List_Of_Teams_By_City.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE traffic.teams_per_city(
+  id BIGSERIAL,
+  city_name VARCHAR(255) NOT NULL,
```
Comment (proulx712) [inline src/main/resources/db/migration/V2__Create_List_Of_Teams_By_City.sql]: This should be a geo code

File: src/main/resources/db/migration/V1__Create_Team_Info_table.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE traffic.team_info(
+  id BIGSERIAL,
+  team_name VARCHAR(255) NOT NULL,
+  schedule_url VARCHAR(255) NOT NULL
+);
```
Comment (proulx712) [inline src/main/resources/db/migration/V1__Create_Team_Info_table.sql:5]: There should be additional fields for each time including which type of league (major, minor, college"

File: src/main/resources/db/migration/V2__Create_List_Of_Teams_By_City.sql (hunk 1 above)
Comment (rwwarren) [reply src/main/resources/db/migration/V2__Create_List_Of_Teams_By_City.sql]: should also have a foreign key instead of the team name

**Comment by rwwarren** [comment]
Should also add DTOs instead of using the model

**Pull request merged**